packages/odyssey-react-mui/src/CheckboxGroup.tsx

```diff
@@ -78,6 +80,7 @@ const CheckboxGroup = ({
       isOptional={!isRequired}
       label={label}
       renderFieldComponent={renderFieldComponent}
+      testId={testId}
```

Comment:
@KevinGhadyani-Okta not sure if this is the right pattern. Right now, for all of the components that use `Field` to wrap the field component I'm applying `testId` to `Field` instead of in the `renderFieldComponent` function.

Reply:
Hmm... That's a good question. I think it makes the most sense to put it in the `renderFieldComponent` function since it's not part of `Field` itself. `Field` technically knows nothing of the element being rendered; only props.

Reply:
@KevinGhadyani-Okta updated just this one and Field for the moment to validate this approach:
I still pass the testId to Field but updated `renderFieldComponent` to take another property in its parameter for dataSe.

You can see it in this commit 0fd57d192b03287a857652a039608f002a36c1aa